pkgs/by-name/la/lazynpm/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "lazynpm";
+  version = "0.1.4";
+
+  src = fetchFromGitHub {
+    owner = "jesseduffield";
+    repo = "lazynpm";
+    tag = "v${version}";
+    hash = "sha256-tMUM9GzzTwp/C9hET301LrdWJzVSMYabSQxM78WdE9M=";
+  };
+
+  vendorHash = null;
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
+    "-X=main.date=1970-01-01T00:00:00Z"
+    "-X=main.buildSource=binaryRelease"
```

```suggestion
    "-X=main.buildSource=nixpkgs"
```

Would probably be better?